src/main/kotlin/com/exactpro/th2/common/schema/cradle/CradleConfiguration.kt

```diff
@@ -65,5 +67,6 @@ data class CradleNonConfidentialConfiguration(
     var timeout: Long = CassandraStorageSettings.DEFAULT_TIMEOUT,
     var pageSize: Int = 5000,
     var cradleMaxEventBatchSize: Long = CassandraStorageSettings.DEFAULT_MAX_EVENT_BATCH_SIZE,
-    var cradleMaxMessageBatchSize: Long = CassandraStorageSettings.DEFAULT_MAX_MESSAGE_BATCH_SIZE
+    var cradleMaxMessageBatchSize: Long = CassandraStorageSettings.DEFAULT_MAX_MESSAGE_BATCH_SIZE,
+    var prepareStorage: Boolean = false
```

Comment:
@MrJonniK @std000 do we want to use the false value as default?

Reply:
> do we want to use the false value as default?

I'd think that storage should be prepared by our "admin tool" by default.
So this option should be false here.

Reply:
I think we should use "false" as default, since in this case it is enough to send "true" only to one box to initialize storage.

Reply:
I think we should not change the default behaviour of estore/mstore/rpt-data-provider. We should change it via the new version of infra which set false to the cradle_manager.json config by default.